pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers = [
 ]
 
 dependencies = [
-    "deadline == 0.49.*",
+    "deadline >= 0.49,< 0.49",
```

Comment:
In a similar PR for KeyShot this is different https://github.com/aws-deadline/deadline-cloud-for-keyshot/pull/138/files :
```
"deadline >= 0.48.9,< 0.50",
```

Reply:
Yea, this should be less than 0.50, since the current proposed change is not a valid specifier.

Reply:
maybe dependabot will fix the formatting in the next try at this